docker-compose-automation.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.8'
+services:
+  consent-postgres:
+    image: postgres:11.6-alpine
+    environment:
+      - POSTGRES_USER=consent
+      - POSTGRES_PASSWORD=consent
```

Comment:
Can we change this to something less realistic-sounding? Something like `automation` or `testing`?